File: problem6.c
// Project Euler problem 5: https://projecteuler.net/problem=5
// Author: Tom Harkness
// May 29, 2021

#include <stdbool.h>
#include <stdio.h>

int main(void) {
    int sumOfSquares = 0;
    int squareOfSums = 0;
    int limit = 100;

    for (int i = 1; i<=limit; i++) {
        sumOfSquares += i*i;
        squareOfSums += i;
    }

    squareOfSums *= squareOfSums;

    printf("%d", squareOfSums-sumOfSquares);
}

File: python/problem6.py
# Project Euler problem 6: https://projecteuler.net/problem=6
# Author: Tom Harkness
# May 29, 2021

if __name__ == "__main__":
    sum_of_squares = sum([x**2 for x in range(1,101)])
    square_of_sums = sum([x for x in range(1,101)])**2

    print(square_of_sums - sum_of_squares)